src/Microsoft.AspNetCore.Blazor.Browser/Rendering/BrowserRenderer.cs

```diff
@@ -18,6 +19,11 @@ public class BrowserRenderer : Renderer, IDisposable
     {
         private readonly int _browserRendererId;
 
+        /// <summary>
```

Comment:
My plan is still to create a subclass of `Renderer` for server-side execution. I haven't gotten to it yet 👍